recipes/hdfs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: hdfs
+  version: "2.0.4"
+
+source:
+  fn: hdfs-2.0.4.tar.gz
+  url: https://pypi.python.org/packages/source/h/hdfs/hdfs-2.0.4.tar.gz
+  md5: c48b86a0fb05d22b5f4eb4cb84fc6452
+
+build:
+  entry_points:
```

Comment:
Just add `script: python setup.py install` here instead of having `build.sh` and `bld.bat` files.